test/setup/wdio.conf.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint camelcase: ["error", {properties: "never"}] */
```

Comment:
Most values here should be derived from the currently globally set `TestEnv` and not set directly in this file. The one exception is `capabilities` (i.e. our Selenium Grid Standalone / Test Browser setup) which largely is configured only here.

Reply:
Did you find this lint instruction to be needed? VS Code didn't object to me at its removal

Reply:
It wasn't new, just how it was before. Good to know it's not needed anymore, removing.